tests/main/users.js

```diff
@@ -0,0 +1,12 @@
+const { expect } = require("chai")
+
+const { client, data } = require("..")
+
+describe("Users", () => {
+    let u
+
+    // it("should have the correct user ID", async () => {
+    //     u = await client.users.get(data.testUser)
```

Comment:
is this the client user? if so, maybe check `client.user.id` as well to make sure the `/users/me` call worked properly

Reply:
I commented it out because there's no user manager yet so I can't do any good checking

Reply:
no user manager? you might need to fetch upstream, [it's been on main for at least 4 months](https://github.com/groupme-js/node-groupme/blame/main/src/managers/UserManager.ts)